### PR TITLE
Add simple filters and ordering

### PR DESCRIPTION
**Pull request by odinuge**

### CODE REVIEW

File: lego/apps/users/views/memberships.py [hunk 1]
```diff
@@ -9,6 +11,12 @@
 class MembershipViewSet(AllowedPermissionsMixin, viewsets.ModelViewSet):
     serializer_class = MembershipSerializer
     filter_class = MembershipFilterSet
+    ordering_fields = ["id", "role", "user__first_name"]
```
Comment (LudvigHz) [inline lego/apps/users/views/memberships.py:14]: Can we rename `user__first_name` to something a bit shorter/human?

**Review by LudvigHz — APPROVED** [review]
lgtm

File: lego/apps/users/views/memberships.py (hunk 1 above)
Comment (odinuge) [reply lego/apps/users/views/memberships.py]: Not gonna use it, so ill just remove it

File: lego/apps/email/filters.py [hunk 2]
```diff
@@ -8,16 +8,30 @@ class EmailListFilterSet(FilterSet):
...
         model = EmailList
-        fields = ("name", "email")
+        fields = ("name", "email", "userFirstName", "userLastName", "userUsername")
 
 
 class EmailUserFilterSet(FilterSet):
 
     email = CharFilter(field_name="internal_email__email", lookup_expr="icontains")
+    userFirstName = CharFilter(
+        field_name="internal_email__user__last_name", lookup_expr="icontains"
```
Comment (odinuge) [inline lego/apps/email/filters.py:28]: ouf

Comment (LudvigHz) [reply lego/apps/email/filters.py:28]: :3rd_place_medal:

**Pull request merged**